src/clients/MpaTabReport.tsx

```diff
@@ -16,8 +21,31 @@ const MpaTabReport = () => {
     { id: representationId, label: t("Representation") },
   ];
   const [tab, setTab] = useState<string>(viabilityId);
+  const [geography, setGeography] = useState("nearshore");
+
+  const geographySwitcher = (e: any) => {
+    setGeography(e.target.value);
+  };
   return (
     <>
+      <Card>
+        <p>
+          {t("Generate reports for the")}{" "}
+          <select onChange={geographySwitcher}>
+            {geographies.map((geography) => {
+              return (
+                <option
+                  key={geography.geographyId}
+                  value={geography.geographyId}
+                >
+                  {geography.display}
+                </option>
+              );
+            })}
+          </select>{" "}
+          {t("planning area.")}
```

Comment:
Consider simplifying this UI messaging to just

`Region: [switcher]`

And trust the user will be told about this feature, and they will understand what is happening when they try it.

Reply:
Terminology does matter, I can see that "planning area" could be instead of `region`

Reply:
Maybe we can chat about it at scrum tomorrow. Then I'll go through and make sure the choice is consistent through the reports.